pyrs990/_options.py

```diff
@@ -116,6 +116,14 @@
     help="file to which output should be written (or ':stdout:', ':stderr:')",
 )
 
+parser.add_argument(
+    "--regions",
+    type=str,
+    default="mt",
+    help="regions to search, comma-separated "
+    + "(two character state abbreviations, for most)",
```

Comment:
What else is there other than state abbreviations? Puerto Rico and DC?

Reply:
International organizations have an abbreviation and then there are regions, like northwest, etc.